src/lib.rs

```diff
@@ -142,7 +159,7 @@
 //! # Configuration
 //!
 //! In order to run the tracker you need to provide the configuration. If you run the tracker without providing the configuration,
-//! the tracker will generate the default configuration the first time you run it. It will generate a `tracker.toml` file with
+//! the tracker will generate the default configuration the first time you run it. It will generate a `Cargo.toml` file with
 //! in the root directory.
```

Comment:
Hi @ivan9989-pkj, this is not true anymore. Now, the tracker uses the default config file `./share/default/config/tracker.development.sqlite3.toml`.

You can use any name for the config file, but we usually use this name `tracker.toml` or `config.toml` but never `Cargo.toml`. That's the Rust crate config file.